File: backend/DvbLiveBackend/ApiStructure/Output/VehiclePosition.cs
using System;
using System.Collections.Generic;

namespace DerMistkaefer.DvbLive.Backend.ApiStructure.Output
{
    /// <summary>
    /// Api Output for an Vehicle Position
    /// </summary>
    public class VehiclePosition
    {
        /// <summary>
        /// Unique Id of this Trip.
        /// </summary>
        public string IdTrip { get; set; } = "";

        /// <summary>
        /// Active Stops of this Trip.
        /// </summary>
        public IEnumerable<VehiclePositionTripStop> Stops { get; set; } = Array.Empty<VehiclePositionTripStop>();
    }
}

File: frontend/src/services/backend/output/VehiclePosition.ts
/**
 * Api Output for an Vehicle Position
 */
import {VehiclePositionTripStop} from "./VehiclePositionTripStop";

export interface VehiclePosition {
    /**
     * Unique Id of this Trip.
     */
    idTrip: string;

    /**
     * Active Stops of this Trip.
     */
    stops: VehiclePositionTripStop[];
}
